presto-main/src/main/java/io/prestosql/sql/planner/LiteralEncoder.java

```diff
@@ -210,6 +217,26 @@ public Expression toExpression(Object object, Type type)
             return new GenericLiteral("DATE", new SqlDate(toIntExact((Long) object)).toString());
         }
 
+        if (type instanceof TimestampWithTimeZoneType) {
+            TimestampWithTimeZoneType timestampType = (TimestampWithTimeZoneType) type;
+            if (timestampType.isShort()) {
+                long epochMillis = unpackMillisUtc((Long) object);
+                ZoneId zoneId = unpackZoneKey((Long) object).getZoneId();
+                return new GenericLiteral("TIMESTAMP(" + timestampType.getPrecision() + ") WITH TIME ZONE", formatTimestampWithTimeZone(
+                        timestampType.getPrecision(),
+                        epochMillis,
+                        0,
+                        zoneId));
+            } else {
```

Comment:
`else` is redundant since the "then" block always returns 
see https://github.com/prestosql/presto/blob/master/DEVELOPMENT.md#additional-ide-configuration